.changeset/open-oranges-sell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": minor
+---
+
+Addressed [#5295](https://github.com/biomejs/biome/issues/5295): option to allow overrides of files.maxSize
```

Comment:
``````suggestion
It's possible to override the option `files.maxSize`. This option is helpful if you need to process specific files that exceed the default `maxSize`:

```json
{
	"overrides": [{
		"includes": ["dist/**"]
		"files": {
			"maxSize": 20000
		}
	}]
}
```
``````